puppet.go

```diff
@@ -22,68 +22,119 @@ import (
 	"io"
 	"log"
 	"os/exec"
+	"strconv"
 	"strings"
+	"time"
 )
 
-func PuppetConfig(puppetParameter, puppetConf string) *GraphConfig {
-	var puppetConfArg string
-	if puppetConf != "" {
-		puppetConfArg = "--config=" + puppetConf
-	}
-
-	var cmd *exec.Cmd
-	if puppetParameter == "agent" {
-		cmd = exec.Command("puppet", "mgmtgraph", "print", puppetConfArg)
-	} else if strings.HasSuffix(puppetParameter, ".pp") {
-		cmd = exec.Command("puppet", "mgmtgraph", "print", puppetConfArg, "--manifest", puppetParameter)
-	} else {
-		cmd = exec.Command("puppet", "mgmtgraph", "print", puppetConfArg, "--code", puppetParameter)
-	}
+const (
+	PUPPET_YAML_BUFFER_SIZE = 65535
```

Comment:
Use golang standard formatting, eg camelCase or CamelCase. So PuppetYAMLBufferSize for example.